src/config/entities/__tests__/configuration.ts

```diff
@@ -270,6 +270,10 @@ export default (): ReturnType<typeof configuration> => ({
   safeWebApp: {
     baseUri: faker.internet.url({ appendSlash: false }),
   },
+  spaces: {
+    maxSafesPerSpace: faker.number.int({ min: 5, max: 10 }),
+    maxInvites: faker.number.int({ min: 5, max: 10 }),
```

Comment:
This is not strictly related with the PR, but I thought it would be nice to group all the spaces-related configuration.